build.gradle

```diff
@@ -28,6 +28,7 @@ dependencies {
 
     testImplementation gradleTestKit()
     testImplementation 'org.spockframework:spock-core:2.3-groovy-3.0'
+    testImplementation 'net.bytebuddy:byte-buddy:1.15.10'
```

Comment:
It's needed to create stubs with Spock from classes (only possible with interfaces otherwise)

Reply:
We could use 
```suggestion
    testImplementation 'net.bytebuddy:byte-buddy:1.15.10' {
        because("It's needed to create stubs with Spock from classes (only possible with interfaces otherwise)")
    }
```
🤓